tensorflow_decision_forests/contrib/temporal_feature_processor/temporal_feature_processor/implementation/pandas/operators/window/tests/simple_moving_average/data/diff_sampling.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test data for SimpleMovingAverage operator."""
+
+import pandas as pd
+
+from temporal_feature_processor.implementation.pandas.data.event import PandasEvent
+from temporal_feature_processor.implementation.pandas.data.sampling import PandasSampling
+
+# set of 4 events with different timestamps for each id
+INPUT = PandasEvent(
```

Comment:
This get me thinking that we have not brainstormed about the pro/con of different time representations. Lets do this next meeting :)

For example, should we allow different timezones or units (https://github.com/pandas-dev/pandas/blob/main/pandas/_libs/tslibs/timestamps.pyx#L129) in the timestamps of a same event. There might be restrictions that will simplify greatly some of the code.

Reply:
Let's definitely discuss this next meeting! We'll do some research until then about timestamp handling in pandas.